### PR TITLE
Update spacing / formatting in some docs

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

1. Ensures spaces between [headers & text](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines) and [lists & text](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md032---lists-should-be-surrounded-by-blank-lines)
2. [Removes any trailing whitespace](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md009---trailing-spaces)
3. [Removes any duplicate empty lines](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md012---multiple-consecutive-blank-lines)
4. Some typo fixes snuck in here
5. Also in a few places I changed list styles between `-` and `*` to match the surrounding lists in the doc

### CODE REVIEW

File: Assets/MixedRealityToolkit.SDK/Profiles/MixedRealityControllerConfigurationProfile.md [hunk 1]
```diff
@@ -93,47 +100,53 @@ The Action each input can perform, is completely up to you.
...
+
 * Position - no change
 * Rotation - X 90 (left) -90 (right), Y 0 , Z 0
 
 ## Additional models available to download
----
 
-### Oculus Hands (provided via the Oculus Opensource project)
-Oculus provide two basic hand models, including animation (although not available through the visualizer currently).  
+### Oculus Hands (provided via the Oculus open source project)
```
Comment (thalbern) [inline Assets/MixedRealityToolkit.SDK/Profiles/MixedRealityControllerConfigurationProfile.md]:
```suggestion
### Oculus hands (provided via the Oculus open source project)
```

File: Assets/MixedRealityToolkit/Providers/Hands/README.md [hunk 2]
```diff
@@ -10,9 +10,10 @@ When enabled, virtual left and/or right hands can be moved in the scene using th
...
 The hand simulation has two modes:
+
   1. Quick mode: Hands are shown only as long as they are moved by the mouse. This mode is useful for simple testing of buttons with a single hand.
   2. Persistent mode: Hands stay visible on the screen, even if they are not moved. This mode is useful for testing two-hand manipulation and accurate placement. Gestures are toggled on/off by mouse clicks.
 
 The simulation mode can be switched for the left/right hand individually by pressing the T/Y keys respectively.
 
-Detailed settings for hand simulation can be found in the SimulatedHandAPI prefab, which is instantiated through the Hand Tracking profile. This includes key bindings for hand movement and mouse button bindings for gestures.
\ No newline at end of file
+Detailed settings for hand simulation can be found in the SimulatedHandAPI prefab, which is instantiated through the Hand Tracking profile. This includes key bindings for hand movement and mouse button bindings for gestures.
```
Comment (thalbern) [inline Assets/MixedRealityToolkit/Providers/Hands/README.md]:
```suggestion
Detailed settings for hand simulation can be found in the SimulatedHandAPI prefab, which is instantiated through the hand tracking profile. This includes key bindings for hand movement and mouse button bindings for gestures.
```

File: Assets/MixedRealityToolkit.SDK/Profiles/MixedRealityControllerConfigurationProfile.md (hunk 1 above)
Comment (keveleigh) [reply Assets/MixedRealityToolkit.SDK/Profiles/MixedRealityControllerConfigurationProfile.md]: I have a follow-up PR I think you're going to like 😉

**Pull request merged**